src/formatter.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const XMLParser = require('./xmlparser/XMLParser');
+const XMLBuilder = require('./xmlbuilder/json2xml');
+
+exports.format = function(xmlString) {
```

Comment:
Discussion points:

1. What is a good name for this function? Do we want to use `XML` prefix for consistency with other exports (e.g. `XMLParser`)?
2. Is it ok to use a function instead of a class? Personally, I prefer to use the simplest solution that works. However, I see how a class-based API can be better for consistency.

    ```js
    const formatter = new XMLFormatter();
    const result = formatter.format('<data>hello</data>');
    ```

3. Do we want to add an `options` argument, to allow callers to tweak certain aspects of parsing & building? Example candidates: `trimValues` and `suppressEmptyNode`?